plugins/plugin-web/che-plugin-web-ext-server/pom.xml

```diff
@@ -57,6 +53,14 @@
             <artifactId>che-plugin-web-ext-shared</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.lsp4j</groupId>
+            <artifactId>org.eclipse.lsp4j</artifactId>
+        </dependency>
```

Comment:
Not sure, it builds just fine, which it normally doesn't when it's not sorted. What's the "good" sort order?